deploy/helm/flightctl/templates/flightctl-default-user-roles.yaml

```diff
@@ -0,0 +1,95 @@
+{{ if eq .Values.global.target "acm" }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: flightctl-admin
+  namespace: {{ default .Release.Namespace .Values.global.auth.k8sRbacNs }}
+rules:
+  - verbs:
+      - '*'
+    apiGroups:
+      - flightctl.io
+    resources:
+      - '*'
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: flightctl-viewer
+  namespace: {{ default .Release.Namespace .Values.global.auth.k8sRbacNs }}
+rules:
+  - verbs:
+      - get
+      - list
+    apiGroups:
+      - flightctl.io
+    resources:
+      - devices
+      - fleets
+      - resourcesyncs
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: flightctl-operator
+  namespace: {{ default .Release.Namespace .Values.global.auth.k8sRbacNs }}
+rules:
+  - verbs:
+      - get
+      - list
+      - create
+      - delete
+      - update
+      - patch
+    apiGroups:
+      - flightctl.io
+    resources:
+      - devices
+      - fleets
+      - resourcesyncs
+  - verbs:
+      - get
+    apiGroups:
+      - flightctl.io
+    resources:
+      - devices/console
+  - verbs:
+      - get
+      - list
+    apiGroups:
+      - flightctl.io
+    resources:
+      - repositories
+
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: flightctl-installer
+  namespace: {{ default .Release.Namespace .Values.global.auth.k8sRbacNs }}
+rules:
+  - verbs:
+      - get
+      - list
+    apiGroups:
+      - flightctl.io
+    resources:
+      - enrollmentrequests
+  - verbs:
+      - post
```

Comment:
Needs to be changed to `put` as result of PR #748.

Reply:
Even though HTTP method changed to `put`, the RBAC verb did not. It is still `post` https://github.com/flightctl/flightctl/blob/main/internal/service/enrollmentrequest.go#L359

So if we won't change it, the role definition is correct.